test/k8sT/manifests/egress-nat-policy.yaml

```diff
@@ -13,3 +13,21 @@ spec:
   destinationCIDRs:
   - INPUT_OUTSIDE_NODE_IP/32
   egressSourceIP: INPUT_EGRESS_IP
+---
+apiVersion: cilium.io/v2alpha1
+kind: CiliumEgressNATPolicy
+metadata:
+  name: egress-to-black-hole
+spec:
+  egress:
+  - podSelector:
+      matchLabels:
+        zgroup: testDSClient
+    namespaceSelector:
+      matchLabels:
+        ns: cilium-test
+  # Route everything to a black hole.
+  # It shouldn't affect in-cluster traffic.
+  destinationCIDRs:
+  - 0.0.0.0/0
+  egressSourceIP: 1.1.1.1 # It's a black hole
```

Comment:
Probably doesn't matter much, but I think using 1.1.1.1 as a black hole is an abuse of the standards. Traffic probably get discarded anyway, but there _is_ something at that address now - Cloudflare's DNS and some web page at least. (But then I don't have a better suggestion.)

Reply:
Sound like a good idea to me

Reply:
Thanks, used 192.0.2.13